src/tests/functional/cfp/test_submission_wizard.py

```diff
@@ -108,6 +111,7 @@ def test_wizard_new_user(self, event, question, client):
         assert user.email == 'testuser@example.org'
         assert user.name == 'Jane Doe'
         assert user.profiles.get(event=event).biography == 'l337 hax0r'
+        assert len(djmail.outbox) == 2  # user email plus orga email
```

Comment:
maybe check the subject so we know the right templates are being used for each mail?